tests/src/Interop/PInvoke/DllImportPath/CMakeLists.txt

```
@@ -5,6 +5,11 @@ include_directories(${INC_PLATFORM_DIR})
 set(SOURCES 
     DllImportPathNative.cpp 
 ) 
+
+# Set CMP0037 to OLD for this directory, so that
+# CMake doesn't complain about the libraries with Unicode characters.
+cmake_policy(SET CMP0037 OLD)
```

Wrap the `cmake_policy` in a CMake version check for a version where the policy exists.

```suggestion
if(NOT (CMAKE_MAJOR_VERSION LESS 3)) 
    cmake_policy(SET CMP0037 OLD)
endif()
```